core/src/main/java/io/substrait/expression/proto/ProtoExpressionConverter.java

```diff
@@ -314,6 +314,8 @@ public Expression.Literal from(io.substrait.proto.Expression.Literal literal) {
           literal.getList().getValuesList().stream()
               .map(this::from)
               .collect(java.util.stream.Collectors.toList()));
+      case EMPTY_LIST -> ExpressionCreator.emptyList(
+          literal.getNullable(), protoTypeConverter.from(literal.getEmptyList().getType()));
```

Comment:
Looking at this helped me identify some potential ambiguity in the spec. There is a nullability field in the `Literal` message AND a nullability field in the `Type.List` message.

Based on how null literals are written about in the spec, I think we should be using the nullability information from the `Type.List`.

I've created a different PR to clear up the spec here: https://github.com/substrait-io/substrait/pull/602



When declaring the

Reply:
> I think you're saying we should be using (2) and ignoring (1) for the nullability of the list literal itself?

Yes, the `Type.List` messages encodes the nullability of both the wrapper and its elements. Using the nullability therein seems consistent with how the nullability for null literals, another type that encodes it's own nullability) is handled.